requirements.txt

```diff
@@ -1,15 +1,15 @@
-appdirs==1.4.4
-asgiref==3.3.1
-black==20.8b1
-click==7.1.2
-Django==3.1.6
-django-environ==0.4.5
-mypy-extensions==0.4.3
-pathspec==0.8.1
-psycopg2-binary==2.8.6
-pytz==2021.1
-regex==2020.11.13
-sqlparse==0.4.1
-toml==0.10.2
-typed-ast==1.4.2
-typing-extensions==3.7.4.3
+appdirs==1.4.4
+asgiref==3.3.1
+black==20.8b1
+click==7.1.2
+Django==3.1.6
+django-environ==0.4.5
+mypy-extensions==0.4.3
+pathspec==0.8.1
+psycopg2-binary==2.8.6
+pytz==2021.1
+regex==2020.11.13
+sqlparse==0.4.1
+toml==0.10.2
+typed-ast==1.4.2
+typing-extensions==3.7.4.3

```

Comment:
i'm not sure why git's catching this as a change (maybe an encoding thing?), but things still work ok locally so this should be fine!